test/options/form.test.js

```diff
@@ -0,0 +1,75 @@
+const assert = require('chai').assert;
+const { Builder, By, until } = require('selenium-webdriver');
+const test = require('selenium-webdriver/testing');
+
+test.describe('Options select form', function() {
+  this.timeout(50000);
+
+  const addr = 'http://localhost:3000';
+  let driver;
+
+  test.beforeEach(() => {
+    driver = new Builder()
+      .forBrowser('chrome')
+      .build();
+  });
+
+  test.afterEach(() => {
+    driver.quit();
+  });
+
+  test.xit('Loads', () => {
+    login(driver);
+    driver.findElement(By.id('logout'))
+      .getText()
+      .then(text => {
+        assert.equal(text, 'Log Out');
+      });
+  });
+
```

Comment:
I think I just want to make more assertions, but all I'm doing is confirming a generic element exists on the page. I suppose I could get more specific with css selectors, but I thought there might be some library I was unaware of. They're ok with me if they look good to you. Thanks!